### PR TITLE
Test option element rendering when a br child is inserted

### PR DESCRIPTION
**Pull request by domenic**
See https://github.com/whatwg/html/issues/2598. Fails in Gecko: filed https://bugzilla.mozilla.org/show_bug.cgi?id=1373474.

### CODE REVIEW

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/243493877)

# Firefox (nightly) #
Testing web-platform-tests at revision 6f2deda1b0674cc2309079f7f06dbd0619e9a2ee
Using browser at version BuildID 20170612100241; SourceStamp 27cad9749cddf68e11fdd4e5d73dad84a8f8cf23
Starting 10 test iterations
All results were stable
## All results ##
<details>
<summary>1 test ran</summary>
<details>
<summary><a href="http://w3c-test.org/submissions/6259/html/rendering/replaced-elements/the-option-element/option-with-br.html">/html/rendering/replaced-elements/the-option-element/option-with-br.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
</details>

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/243493878)

# Sauce (safari) #
Testing web-platform-tests at revision 6f2deda1b0674cc2309079f7f06dbd0619e9a2ee
Using browser at version 10.0
Starting 10 test iterations
All results were stable
## All results ##
<details>
<summary>1 test ran</summary>
<details>
<summary><a href="http://w3c-test.org/submissions/6259/html/rendering/replaced-elements/the-option-element/option-with-br.html">/html/rendering/replaced-elements/the-option-element/option-with-br.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | PASS    |          |

</details>
</details>

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/243493887)

# Chrome (unstable) #
Testing web-platform-tests at revision 6f2deda1b0674cc2309079f7f06dbd0619e9a2ee

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/243493888)

# Sauce (MicrosoftEdge) #
Testing web-platform-tests at revision 6f2deda1b0674cc2309079f7f06dbd0619e9a2ee

File: html/rendering/replaced-elements/the-option-element/option-with-br-ref.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>option element with br child</title>
+<link rel="author" title="Domenic Denicola" href="mailto:d@domenic.me">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#concept-option-label">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#dom-option-text">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#the-select-element-2">
+
+<link rel="match" href="option-with-br-ref.html">
```
Comment (phistuck) [inline html/rendering/replaced-elements/the-option-element/option-with-br-ref.html]: Should be -
`<link rel="match" href="option-with-br.html">`

Comment (domenic) [inline html/rendering/replaced-elements/the-option-element/option-with-br-ref.html]: No, it's a no-op in match files. It should probably just be deleted, but it definitely shouldn't be circular.

**Review by phistuck — APPROVED** [review]

**Review by zcorpan — APPROVED** [review]

**Pull request merged**